setup.py

```diff
@@ -37,6 +37,7 @@
     install_requires=[
         "Adafruit-Blinka",
         "adafruit-circuitpython-requests",
+        "adafruit-requests",
```

Comment:
You can remove this, the package you're trying to import is imported just above this!